src/style.css

```diff
@@ -0,0 +1,90 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 300;
+}
+
+input {
+  display: block;
+  width: 100%;
+  height: 100%;
+  font-family: inherit;
+  font-weight: inherit;
+}
+
+input:focus::placeholder {
+  color: transparent;
+}
```

Comment:
use class instead tag. By the way you have it

Reply:
I don't understand why error still appears? I need you help to figure this out